src/lib/Event/Subscriber/TrashEventSubscriber.php

```diff
@@ -8,14 +8,36 @@
 
 namespace EzSystems\EzRecommendationClient\Event\Subscriber;
 
+use eZ\Publish\API\Repository\ContentService as ContentServiceInterface;
 use eZ\Publish\API\Repository\Events\Trash\RecoverEvent;
 use eZ\Publish\API\Repository\Events\Trash\TrashEvent;
+use eZ\Publish\API\Repository\LocationService as LocationServiceInterface;
```

Comment:
I don't see `Core\...\LocationService` used in this file. Why was this aliased?

Reply:
Same goes for `ContentService`.

Reply:
It comes from AbstractRepositoryEventSubscriber

Reply:
Then that comment is also valid for `AbstractRepositoryEventSubscriber`, OTOH I do not see how aliasing comes from parent class. In Both you could just use `use eZ\Publish\API\Repository\ContentService`

Reply:
I will remove aliasing in next PR. This PR is very important to make Behat test green